### PR TITLE
add development docs for new engineers: NET-1482

### PR DESCRIPTION
**Pull request by gvelez17**
## Description

Provide Alex's postman sample queries and configuration for new developers to set up webstorm and test

## How Has This Been Tested?

Describe the tests that you ran to verify your changes. Provide instructions for reproduction.

- [ ] Test A (e.g. Test A - New test that ... ran in local, docker, and dev unstable.)
- [ ] Test B

## PR checklist

Before submitting this PR, please make sure:

- [ ] I have tagged the relevant reviewers and interested parties
- [ ] I have updated the READMEs of affected packages
- [ ] I have made corresponding changes to the documentation

## References:

Please list relevant documentation (e.g. tech specs, articles, related work etc.) relevant to this change, and note if the documentation has been updated.

### CODE REVIEW

**Comment by willex** [comment]
@ukstv - could you verify that the package descriptions are accurate or if you would change anything
@mattdavis0351 - feel free to review & add/change anything you'd like to

**Review by willex — APPROVED** [review]

File: docs-dev/DEVELOPMENT_LOCAL.md
Comment (gvelez17) [inline docs-dev/DEVELOPMENT_LOCAL.md:81]: what does this mean?

File: docs-dev/DEVELOPMENT_LOCAL.md
Comment (gvelez17) [inline docs-dev/DEVELOPMENT_LOCAL.md:89]: not sure how this works, link?

**Comment by gvelez17** [comment]
hey @willex this is a great improvement but before we expose it to external/newbie devs I think we should walk thru and make sure we're communicating the core concepts meaningfully - noted a few places it is not clear to me how to better describe those dirs.  I'd like to wait for the reviews you asked for before merging, or we can jump on and try to hammer it out more clearly.

File: docs-dev/DEVELOPMENT_LOCAL.md
Comment (ukstv) [reply docs-dev/DEVELOPMENT_LOCAL.md:96]: mechanism to safely bootstrap connections to other ceramic nodes on the network.

File: docs-dev/DEVELOPMENT_LOCAL.md
Comment (ukstv) [reply docs-dev/DEVELOPMENT_LOCAL.md:104]: Makes multiple pinning backends appear as one. Allows one to pin data to multiple backends (IPFS, Arweave, Crus, Filecoin,...) simultaneously.

File: docs-dev/DEVELOPMENT_LOCAL.md
Comment (ukstv) [reply docs-dev/DEVELOPMENT_LOCAL.md:111]: [Tile streamtype](https://developers.ceramic.network/docs/advanced/standards/stream-programs/tile-document/) - stores a mutable JSON document with schema validation, providing similar functionality as a NoSQL document.

**Comment by ukstv** [comment]
One additional thing here. Please, @gvelez17 mention that a developer should use Node v16, and npm v8.

For switching between node versions one could use [n](https://github.com/tj/n), [fnm](https://github.com/Schniz/fnm) or [nvm](https://github.com/nvm-sh/nvm)

**Review by ukstv — CHANGES_REQUESTED** [review]
Looks really nice. Needs a note about Node v16 and npm v8.

File: docs-dev/DEVELOPMENT_LOCAL.md
Comment (mattdavis0351) [inline docs-dev/DEVELOPMENT_LOCAL.md:34]: This assumes too much about the developers environment.  There is a possibility that they are not using ZSH as their shell, which makes this command not work for anyone not using ZSH.

File: docs-dev/DEVELOPMENT_LOCAL.md
Comment (mattdavis0351) [inline docs-dev/DEVELOPMENT_LOCAL.md:32]: Consider stating which versions of Python are needed, rather than saying `not python3`.

I'm also struggling to see why I need a Python Environment.  Is ti needed for WebStorm to run properly?  What is the purpose of this step?

Comment (gvelez17) [reply docs-dev/DEVELOPMENT_LOCAL.md:32]: I am not clear on it - @willex ?

Comment (willex) [reply docs-dev/DEVELOPMENT_LOCAL.md:32]: Yeah, going to update it. By default there is no `python` alias available when using a vanilla MacOS install which makes the standard npm install fail (just ran into that issue with Artur setting up his new laptop).

File: docs-dev/DEVELOPMENT_LOCAL.md
Comment (willex) [reply docs-dev/DEVELOPMENT_LOCAL.md:104]: I've added @ukstv explanation next to aggregation.

File: docs-dev/DEVELOPMENT_LOCAL.md
Comment (willex) [reply docs-dev/DEVELOPMENT_LOCAL.md:96]: added @ukstv to package explanation.

File: docs-dev/DEVELOPMENT_LOCAL.md
Comment (willex) [reply docs-dev/DEVELOPMENT_LOCAL.md:111]: Updated @ukstv description

**Pull request merged**
